OracleEDQ/12.2.1.3.0/container-scripts/CreateEDQDomain.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+# Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Script to make simple EDQ 12.2.1 domain
+# ---------------------------------------
+execfile("/u01/oracle/container-scripts/edq.py")
+i = 1
+while i < len(sys.argv):
```

Comment:
You really should be using `argparse` for this and not a huge `while` block: https://docs.python.org/2.7/library/argparse.html

The `argparse` module is available by default in the `oraclelinux:7-slim` container.

Reply:
Used OracleFMWInfrastructure scripts as reference to create EDQ docker scripts. So used while block.